Sources/ViewModels/ALKConversationViewModel.swift

```diff
@@ -458,6 +458,8 @@ open class ALKConversationViewModel: NSObject, Localizable {
             }
         case .form:
             return 0
+        case .initialEncryptionMessage:
```

Comment:
Should it be `initialEncryptionMessage` when it's static top message in other places?

Reply:
changed top statictopmessage